File: Conceitos de LPs/Tipos de Dados/Cadeia de Caracteres/cadeiacaracteres.php
<?php
	//PHP
    $str1 = "Olá";
    $str2 = "Mundo";

    // Concatenar
    $concatenado = $str1 . " " . $str2;
    echo "Concatenado: $concatenado\n";

    // Substring
    $sub = substr($concatenado, 0, 3);
    echo "Substring: $sub\n";

    // Comparar strings
    $iguais = ($str1 === $str2);
    echo "As strings são iguais? " . ($iguais ? 'sim' : 'não') . "\n";

    // Tamanho da string
    echo "Tamanho de str1: " . strlen($str1) . "\n";
?>

File: Conceitos de LPs/Tipos de Dados/Cadeia de Caracteres/CadeiaCaracteres.java
//Java
public class CadeiaCaracteres {
    public static void main(String[] args) {
        String str1 = "Olá";
        String str2 = "Mundo";

        // Concatenar
        String concatenado = str1 + " " + str2;
        System.out.println("Concatenado: " + concatenado);

        // Substring
        String sub = concatenado.substring(0, 3);
        System.out.println("Substring: " + sub);

        // Comparar strings
        boolean iguais = str1.equals(str2);
        System.out.println("As strings são iguais? " + iguais);

        // Tamanho da string
        System.out.println("Tamanho de str1: " + str1.length());
    }
}
